docs/dev/libvirt-howto.md

```diff
@@ -238,7 +242,7 @@ Using the domain names above will only work if you [set up the DNS overlay](#set
 Alternatively, if you didn't set up DNS on the host, you can use:
 
 ```sh
-virsh domifaddr master0  # to get the master IP
+sudo virsh domifaddr master0  # to get the master IP
```

Comment:
An alternative here is to explicitly set the same connection parameters you configured for the cluster.  For example, after [setting the environment variable][1], use:

```console
$ virsh -c "${OPENSHIFT_INSTALL_LIBVIRT_URI}" domifaddr master0
```

That will work without local privilege escalation, because the cluster-API operator is using the same approach to access libvirt, and it's not using `sudo` on your host ;).

[1]: https://github.com/openshift/installer/blob/ace571b041049c276443631f349dbd9e0dad27e5/docs/dev/libvirt-howto.md#build-and-run-the-installer

Reply:
@wking I do like that alternative, actually this is more accurate.